cumulusci.yml

```diff
@@ -340,22 +340,30 @@ tasks:
             path: scripts/configure_npsp_default_settings.cls
             apex: initializeNPSPSettings();
 
-    enable_customizable_rollups:
-        description: Enable the NPSP Customizable Rollups feature (unmanaged deploys only)
+    enable_crlp:
```

Comment:
There is one mention of the old name `enable_customizable_rollups` in the codebase, so that should be renamed as well. Is there any `cci` documentation and similar that need to be updated due to the task name change?

Reply:
Thanks for the reminder!